helm_deploy/values-dev.yaml

```diff
@@ -24,3 +24,4 @@ env:
   COMMUNITYAPI_BASEURL: "https://community-api-secure.test.delius.probation.hmpps.dsd.io"
   ASSESSRISKSANDNEEDS_BASEURL: "https://assess-risks-and-needs-dev.hmpps.service.justice.gov.uk"
   SENTRY_ENVIRONMENT: "dev"
+  AWS_S3_ENABLED: "true"
```

Comment:
Where is this variable used?
I can only see that `aws.s3.enabled` is used but from the application.yaml....

Reply:
these env vars and the configuration file entries are interchangeable. 

it's the same with the env vars like `COMMUNITYAPI_BASEURL` which translates to `community-api.baseurl`